driver/pparse.ml

```diff
@@ -166,6 +166,9 @@ let file ppf ~tool_name inputfile parse_fun ast_magic =
   close_in ic;
   apply_rewriters ~restore:false ~tool_name ast_magic ast
 
+let file ppf ~tool_name inputfile parse_fun ast_magic =
+  Timings.(time (Parsing inputfile))
+    (file ppf ~tool_name inputfile parse_fun) ast_magic
```

Comment:
I'm not fond of the way you mix different levels of abstraction here, with *most* `Timings` call placed in pass-management code, and this one deep down inside the `Pparse` auxiliary functions. I think the code would actually be easier to read and maintain if you deduplicated those `Timings.time` call to the appropriate callers of `Pparse` functions.